test/e2e/page-objects/pages/dialog/snap-install.ts

```diff
@@ -12,8 +13,24 @@ class SnapInstall {
 
   private readonly scrollSnapInstall = '[data-testid="snap-install-scroll"]';
 
+  private readonly scrollSnapUpdate = '[data-testid="snap-update-scroll"]';
+
   private readonly approveButton = '[data-testid="confirmation-submit-button"]';
 
+  private readonly connectButton = '[data-testid="confirm-btn"]';
+
+  public readonly messageLifeCycleHookSpan = '.snap-ui-renderer__panel';
```

Comment:
I think this is actually a `div`, perhaps `lifeCycleHookMessageElement` is a bit better?